src/__test__/utils/cellSets/getAllCells.test.js

```diff
@@ -25,7 +25,7 @@ describe('Get all cells', () => {
 
     expect(allCellIds[0]).toMatchInlineSnapshot(`
       Object {
-        "cellId": 11,
+        "cellId": 1,
```

Comment:
I added cells and made changes to the mock `__test__/data/cell_sets.json` file to be able to test the check introduced here. If you look at the diff for [cell_sets.json](https://github.com/hms-dbmi-cellenics/ui/pull/650/files#diff-4a1bbbf10ce496597b0ecff72b7426bc5942ba042d604319475c5110650c41d1R145-R153), the first cell id of sample `"b62028a1-ffa0-4f10-823d-93c9ddb88898"` was initiallly `11`.  I rearranged the cells in the sample and it was changed to 1.

Reply:
Woah, I would have though that changing cell_sets.json would have caused the need for way more changes in unit tests since it is being used in virtually all of the new unit tests.

Next time, however, if you need a new dataset I think I would recommend just creating a different `.json` for your use case instead of changing an existing one and having to update a lot of different tests.

Reply:
I think sometihng good about the previous `cell_sets.json` was that it was simple so it was easy to understand the tests we were making with it

Reply:
this is also simple - there's just more cells